packages/sitecore-jss-nextjs/src/components/NextImage.tsx

```diff
@@ -92,7 +84,7 @@ export const NextImage: React.FC<NextImageProps> = ({
     delete imageProps.height;
   }
 
-  const loader = (otherProps.loader ? otherProps.loader : sitecoreLoader) as ImageLoader;
+  const loader = (otherProps.loader ? otherProps.loader : undefined ) as ImageLoader;
```

Comment:
not sure if we need it but maybe a comment mentioning that if we have custom loader it will override vercel's built in image optimization?

Reply:
Removed the loader prop since it is already part of the imageProps.